### DISCUSSION TITLE
Frigate+ annotatios: hints for two different scenarios

### DISCUSSION THREAD

**Discussion by JanickGers85** [blakeblackshear/frigate#10013] [Ask A Question] [answered]
Hello there! Happy Frigate+ customer here, but I'm asking some clarification about annotation of images.
More specifically I'm in doubt on how to label those:

1. Bicycles
![image](https://github.com/blakeblackshear/frigate/assets/21289907/08f2db71-2b72-4f36-a61f-5c21d02e8b76)
Since I switched to payed models they get recognised as person with a lower confidence of the persons not riding a bicycle.
Frigate+ do not permit to annotate bicycles; how to proceed?
* avoid sending any image that contains at least one bicycles since Frigate+ do not permit to label it and the model can be confused
* send the image anyway and label that as a person
2. Persons with umbrella
![image](https://github.com/blakeblackshear/frigate/assets/21289907/a99159fc-249b-4d09-aaff-e545f319de31)
When labeling this, do I need to include the whole person and umbrella in the box, or I have to draw a box containing only the shape of the person (like we do when there's an obstacle?)

**Comment by NickM-27** [comment, marked as answer]
1. just send the image and annotate all of the labels that are available in frigate+, there is no reason to avoid submitting this image
2. the full umbrella should not be included, I believe the recommendation is to label the person (including parts occluded by the umbrella)

**Reply by JanickGers85** [reply]
@NickM-27 Thanks for the responses!
Just one last question: do I have to label the person on bicycles as persons or I have to ignore them?

**Reply by NickM-27** [reply]
Yes you should label it
